files/pt-br/orphaned/glossary/dynamic_programming_language/index.md

```diff
@@ -14,4 +14,4 @@ Isso é o oposto das linguagens denominadas **linguagens de programação estát
 
 ### Conhecimento geral
 
-- {{Interwiki("wikipedia", "Linguagem de Programação Dinâmica")}} on Wikipedia
+- [Linguagem de Programação Dinâmica](https://pt.wikipedia.org/wiki/Linguagem_de_Programação_Dinâmica) on Wikipedia
```

Comment:
This URL does not exist. Should we remove it?

Reply:
Yes, we could remove this (actually, this document [should be removed](https://github.com/mdn/translated-content/blob/main/docs/orphaned_and_conflicting.md#orphaned-pages))